tests/phpunit/tests/rest-api/rest-controller.php

```diff
@@ -300,6 +300,22 @@ public function test_get_endpoint_args_for_item_schema() {
 		$this->assertArrayHasKey( 'someobject', $args );
 	}
 
+	public function test_get_endpoint_args_for_item_schema_validate_callback() {
+		$controller = new WP_REST_Test_Controller();
+		$args       = rest_get_endpoint_args_for_schema( $controller->get_item_schema() );
+
+		$this->assertSame( $args['somestring']['validate_callback'], '__return_true' );
+		$this->assertSame( $args['sometextfield']['validate_callback'], 'rest_validate_request_arg' );
```

Comment:
```suggestion
		$this->assertSame( '__return_true', $args['somestring']['validate_callback'] );
		$this->assertSame( 'rest_validate_request_arg', $args['sometextfield']['validate_callback'] );
```
Expected value should be first.